src/test/java/testsuite/failover/FailoverIntegrationTest.java

```diff
@@ -610,72 +650,47 @@ public void test3_4_writerFailWithNoTransaction() throws SQLException, Interrupt
   public void test4_1_pooledWriterConnection_basicfailover()
       throws SQLException, InterruptedException {
     final String initalWriterId = getDBClusterWriterInstanceId();
-    assertEquals(INSTANCE_ID_1, initalWriterId);
+    assertEquals(instanceIDs[0], initalWriterId);
 
     testConnection = createPooledConnectionWithInstanceId(initalWriterId);
 
     // Crash writer Instance1 and nominate Instance2 as the new writer
-    failoverClusterToATargetAndWaitUntilWriterChanged(initalWriterId, INSTANCE_ID_2);
+    failoverClusterToATargetAndWaitUntilWriterChanged(initalWriterId, instanceIDs[1]);
 
     assertFirstQueryThrows(testConnection, "08S02");
 
     // Execute Query again to get the current connection id;
     final String currentConnectionId = queryInstanceId(testConnection);
+
     // Assert that we are connected to the new writer after failover happens.
     assertTrue(isDBInstanceWriter(currentConnectionId));
     final String nextWriterId = getDBClusterWriterInstanceId();
     assertEquals(nextWriterId, currentConnectionId);
-    assertEquals(INSTANCE_ID_2, currentConnectionId);
+    assertEquals(instanceIDs[1], currentConnectionId);
 
     // Assert that the pooled connection is valid.
     assertTrue(testConnection.isValid(IS_VALID_TIMEOUT));
-
-    // Start crashing all other instances
-    startCrashingInstanceAndWaitUntilDown(INSTANCE_ID_3);
```

Comment:
are these chunks just not necessary anymore?

Reply:
No, they're replaced by another set of methods that crash/check nodes in parallel.

Reply:
Right but is the assertFalse(testConnection.isValid(IS_VALID_TIMEOUT)) important to test still or no?

Reply:
followed up offline, wasn't necessary